openstack/designate/templates/producer-deployment.yaml

```diff
@@ -75,18 +75,16 @@ spec:
           livenessProbe:
             exec:
               command:
```

Comment:
memcached as a dependency was needed for coordination before we switched to file-based coordination